src/pages/Counter/components/CounterView/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
```

Comment:
imports from libs and from your local files must be separated with a space line. Fix it in all places